DXRCore/DeusEx/Classes/DXRActorsBase.uc

```diff
@@ -258,13 +283,16 @@ static function Inventory GiveExistingItem(Pawn p, Inventory item, optional int
         a = Ammo(p.FindInventoryType(item.class));
         if( a != None ) {
             a.AmmoAmount += a.default.AmmoAmount + add_ammo;
+            a.AmmoAmount = min(a.AmmoAmount, a.MaxAmmo);
```

Comment:
actually this change might be bad, limiting ammo for enemies in high loops of WW?